src/less/filter-menu-button/base/filter-menu-button.less

```diff
@@ -0,0 +1,247 @@
+    min-width: 144px;
+    overflow: hidden;
+    position: absolute;
+    top: calc(100% + 8px);
+    width: max-content;
+    z-index: 1;
+}
+
+button.filter-menu-button__button[aria-expanded="true"] + .filter-menu-button__menu {
+    display: block;
+}
+
+.filter-menu-button__items {
+    margin-top: 8px;
+    max-height: calc(50vh - 40px); // half of the viewport height || TODO remove max-height restriction to filter-menu styles
+    min-width: 100%;
+    overflow-y: auto;
+    position: relative;
+}
+
+span.filter-menu-button__items {
+    display: inline-block;
+}
+
+span.filter-menu-button__status,
+span.filter-menu-button__status--inline {
+    display: block;
+    flex-shrink: 1;
+    height: 18px;
+    min-width: 18px;
+    width: 18px;
+}
+
+span.filter-menu-button__status {
+    .background-icon-base-mixin();
+    .background-icon-image-mixin('checkbox-unchecked', 18px, 18px);
+}
+
+span.filter-menu-button__status--inline {
+    svg {
+        &.icon.icon--checkbox-unchecked {
+            color: @filter-menu-checkbox-color;
+            display: block;
+            height: 18px;
+            width: 18px;
+        }
+
+        &.icon.icon--checkbox-checked {
+            color: @filter-menu-checkbox-checked-color;
+            display: none;
+            height: 18px;
+            width: 18px;
+        }
+    }
+}
+
+.filter-menu-button__item {
+    .filter-menu-item-base();
+
+    padding: 8px 16px;
+
+    &:focus {
+        outline: none;
+    }
+
+    &:focus,
+    &:hover {
+        background-color: @filter-menu-item-focus-background-color;
+    }
+
+    &:last-child {
+        margin-bottom: 8px;
+    }
+
+    & > span:not(.checkbox):not([class*="--inline"]) {
+        flex-grow: 1;
+        margin-left: 8px;
+    }
+
+    span.checkbox {
+        align-items: flex-start;
+        display: flex;
+        flex-shrink: 1;
+        justify-content: center;
+
+        svg {
+            &.checkbox__unchecked {
+                color: @filter-menu-checkbox-color;
+            }
+
+            &.checkbox__checked {
+                color: @filter-menu-checkbox-checked-color;
+            }
+        }
+    }
+}
+
+span.filter-menu-button__item {
+    display: block;
+}
+
+button.filter-menu-button__footer {
+    .filter-menu-item-base();
+
+    border: none;
+    border-top: 1px solid @filter-menu-border-color;
+    bottom: 0;
+    padding: 16px;
+
+    &:focus {
+        outline: none;
+    }
+
+    &:focus,
+    &:hover {
+        background-color: @filter-menu-item-focus-background-color;
+    }
+}
+
+.filter-menu-button__text {
+    flex-grow: 1;
+    margin-left: 8px;
+}
+
+.filter-menu-button__item[role="menuitemcheckbox"][aria-checked="true"] > span.filter-menu-button__status {
+    .background-icon-image-mixin('checkbox-checked', 18px, 18px);
+}
+
+.filter-menu-button__item[role="menuitemcheckbox"][aria-checked="true"] > span.filter-menu-button__status--inline > svg.icon.icon--checkbox-unchecked {
```

Comment:
(minor) This and the next selector have a lot in common except for the last part.  They could be combined.

Reply:
@msendlakowski You're proposing something like this?:

```
.filter-menu-button__item[role="menuitemcheckbox"][aria-checked="true"]>span.filter-menu-button__status--inline>svg.icon.icon--checkbox {
    &-unchecked {
        display: none;
    }

    &-checked {
        display: block;
    }
}
```

Reply:
No, cause I'd be reprimanded for that.  I'm thinking:
```
.filter-menu-button__item[role="menuitemcheckbox"][aria-checked="true"]>span.filter-menu-button__status--inline {
    > svg.icon.icon--checkbox-unchecked {
        display: none;
    }

    > svg.icon.icon--checkbox-checked {
        display: block;
    }
}
```

Reply:
Ha ha. Yeah, that makes sense.